README.md

```diff
@@ -61,7 +61,7 @@ To setup an admin user on mongodb:
 
     Enter the mongo shell
     use admin
-    db.addUser( { user: "admin", pwd: "password", roles: [ "clusterAdmin", "readWriteAnyDatabase", "userAdminAnyDatabase", "dbAdminAnyDatabase"] } )
+    db.createUser( { user: "admin", pwd: "password", roles: [ "clusterAdmin", "readWriteAnyDatabase", "userAdminAnyDatabase", "dbAdminAnyDatabase"] } )
```

Comment:
Which version of MongoDB are you using? Your fix is most likely correct, and if so, we need to update it across all of our projects. I'm on MongoDB 2.4 at home and at work. I don't know if Bedrock works with MongoDB 2.6 yet.

/cc @dlongley